content/22.vaccines.md

```diff
@@ -327,6 +356,14 @@ All moderate to severe cases of COVID-19 occurred in the placebo group.
 Additionally, in both trials, the vaccine was found to be well-tolerated [@doi:10.1101/2021.10.05.21264567; @doi:10.1056/NEJMoa2107659].
 The conclusions of both trials indicate that the NVX-CoV2373 vaccine is safe and effective against COVID-19, and those who received this vaccine through the trials are considered fully vaccinated [@url:https://ir.novavax.com/Novavax-Statement-on-UK-and-Mexico-Phase-3-Clinical-Trial-Participants-Considered-Fully-Vaccinated-in-the-US].
 
+Similarly, Covifenz was reported to be 71% effective in preventing COVID-19 in the per-protocol analysis [@doi:10.1056/NEJMoa2201300].
+Efficacy was only slightly slower in the intention-to-treat group at 69%.
+Prevention of moderate-to-severe disease was estimated at 78.8% in the intention-to-treat group.
+Over 24,000 participants were included in the safety analysis, which reported that 92.3% of vaccine recipients reported local adverse events compared to 45.5% of placebo recipients, with rates for systemic adverse events at 87.3% and 65.0%, respectively.
+However, the AEs reported were generally mild to moderate.
```

Comment:
Are we abbreviating to AEs everywhere?

Reply:
I'm going to do that on my next read through, somehow we need to cut a lot of words still, so that's one way, haha

Reply:
Ha! Very good